### PR TITLE
fix broken link to htdp 2e

### PR DESCRIPTION
**Pull request by spdegabrielle**
http://www.ccs.neu.edu/home/matthias/HtDP2e/i1-2.html is now 
https://htdp.org/2018-01-06/Book/i1-2.html

reported by Alice on slack https://racket.slack.com/archives/C06V96CKX/p1592042294076100

### CODE REVIEW

File: htdp-doc/scribblings/htdp-langs/common.rkt [hunk 1]
```diff
@@ -16,7 +16,7 @@
 @; -----------------------------------------------------------------------------
 (define dots (bold "..."))
 
-(define htdp "http://www.ccs.neu.edu/home/matthias/HtDP2e/")
+(define htdp "https://htdp.org/2018-01-06/Book/")
```
Comment (mfelleisen) [inline htdp-doc/scribblings/htdp-langs/common.rkt:19]: This points to an old release of "2e". Let's just point to HtDP.org please.

Comment (spdegabrielle) [reply htdp-doc/scribblings/htdp-langs/common.rkt]: apologies now https://htdp.org/2020-5-6/Book/

**Comment by mfelleisen** [comment]
[No apologies needed.]


I may release another one tomorrow Is it important to point to the inside of the book as opposed to https://htdp.org <https://htdp.org/> only? (Sorry I lost track of the context.)

**Comment by spdegabrielle** [comment]
Inside - Alice  noticed the link to Intermezzo 1 was broken in https://docs.racket-lang.org/htdp-langs/index.html but I'm sure it would be fine to wait a day. 

s.

> Alice  10:58 AM
> The first link on https://docs.racket-lang.org/htdp-langs/beginner.html is currently broken. A working link to that passage is here https://htdp.org/2018-01-06/Book/i1-2.html

**Comment by mfelleisen** [comment]
Sorry I didn’t literally mean “tomorrow” but in the sense of American English “some time down the line.” Go push this thing in. Thanks.

**Comment by spdegabrielle** [comment]
More confusion [rofl] 
> Go push this thing in. 

I can't push it.

**Comment by spdegabrielle** [comment]
you my also prefer to `squash` as I haven't worked out how to do that yet.

**Pull request merged**